docs/core/development/policy/coding-standard.rst

```diff
@@ -625,6 +625,126 @@ Any attribute, method argument, or method name that corresponds *directly* to a
 All SQL keywords should be in upper case.
 
 
+Emitting logs
+-------------
+
+All new code emitting log messages should use the new logging system.
```

Comment:
Rather than call it "the new logging system", we should refer to it as `twisted.logger`.  This way the doc will read correctly even after `twisted.python.log` is but a distant regret.  (Hopefully, in our time :))